[CI] :x-pack:plugin:sql:qa:jdbc:security:with-ssl:v7.10.1#bwcTest (fips) failure

**Build scan**: https://gradle-enterprise.elastic.co/s/elzmdxfctoyi6
 (ES_RUNTIME_JAVA=zulu11,nodes=general-purpose)

**Repro line**: (no reproduce line)

**Reproduces locally?**: No

**Applicable branches**: master

**Failure history**:

* https://gradle-enterprise.elastic.co/s/3cmfp5x4tcmeg (ES_RUNTIME_JAVA=zulu11,nodes=general-purpose)
* https://gradle-enterprise.elastic.co/s/5vyjqckrl3v5g (ES_RUNTIME_JAVA=java11,nodes=general-purpose)
* https://gradle-enterprise.elastic.co/s/yrwejit7yf3h6 (ES_RUNTIME_JAVA=adoptopenjdk11,nodes=general-purpose)
* https://gradle-enterprise.elastic.co/s/gvpm7bmsp4kzk (ES_RUNTIME_JAVA=zulu8,nodes=general-purpose)

Seems to be fips related.

**Failure excerpt**:

QA module can't start node and build fails:

```
ERROR][o.e.b.ElasticsearchUncaughtExceptionHandler] [v7.10.1-bwcTest-0] uncaught exception in thread [main] |  
-- | --
  | »  org.elasticsearch.bootstrap.StartupException: org.elasticsearch.ElasticsearchSecurityException: failed to load SSL configuration [xpack.security.transport.ssl] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:174) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:161) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:86) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:127) ~[elasticsearch-cli-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.cli.Command.main(Command.java:90) ~[elasticsearch-cli-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:126) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:92) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  Caused by: org.elasticsearch.ElasticsearchSecurityException: failed to load SSL configuration [xpack.security.transport.ssl] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.lambda$loadSSLConfigurations$5(SSLService.java:533) ~[?:?] |  
  | »  	at java.util.HashMap.forEach(HashMap.java:1336) ~[?:?] |  
  | »  	at java.util.Collections$UnmodifiableMap.forEach(Collections.java:1505) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.loadSSLConfigurations(SSLService.java:529) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.<init>(SSLService.java:140) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createSSLService(XPackPlugin.java:413) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createComponents(XPackPlugin.java:263) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.lambda$new$15(Node.java:542) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:546) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:279) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:170) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	... 6 more |  
  | »  Caused by: org.elasticsearch.ElasticsearchException: failed to create trust manager |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig$CombiningTrustConfig.createTrustManager(TrustConfig.java:171) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.createSslContext(SSLService.java:442) ~[?:?] |  
  | »  	at java.util.HashMap.computeIfAbsent(HashMap.java:1133) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.lambda$loadSSLConfigurations$5(SSLService.java:531) ~[?:?] |  
  | »  	at java.util.HashMap.forEach(HashMap.java:1336) ~[?:?] |  
  | »  	at java.util.Collections$UnmodifiableMap.forEach(Collections.java:1505) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.loadSSLConfigurations(SSLService.java:529) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.<init>(SSLService.java:140) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createSSLService(XPackPlugin.java:413) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createComponents(XPackPlugin.java:263) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.lambda$new$15(Node.java:542) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:546) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:279) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:170) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	... 6 more |  
  | »  Caused by: org.elasticsearch.ElasticsearchException: failed to initialize SSL TrustManager |  
  | »  	at org.elasticsearch.xpack.core.ssl.StoreKeyConfig.createTrustManager(StoreKeyConfig.java:103) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig$CombiningTrustConfig.lambda$createTrustManager$0(TrustConfig.java:168) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:550) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:517) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig$CombiningTrustConfig.createTrustManager(TrustConfig.java:169) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.createSslContext(SSLService.java:442) ~[?:?] |  
  | »  	at java.util.HashMap.computeIfAbsent(HashMap.java:1133) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.lambda$loadSSLConfigurations$5(SSLService.java:531) ~[?:?] |  
  | »  	at java.util.HashMap.forEach(HashMap.java:1336) ~[?:?] |  
  | »  	at java.util.Collections$UnmodifiableMap.forEach(Collections.java:1505) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.loadSSLConfigurations(SSLService.java:529) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.<init>(SSLService.java:140) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createSSLService(XPackPlugin.java:413) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createComponents(XPackPlugin.java:263) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.lambda$new$15(Node.java:542) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:546) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:279) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:170) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	... 6 more |  
  | »  Caused by: java.io.IOException: Invalid keystore format |  
  | »  	at sun.security.provider.JavaKeyStore.engineLoad(JavaKeyStore.java:667) ~[?:?] |  
  | »  	at sun.security.util.KeyStoreDelegator.engineLoad(KeyStoreDelegator.java:222) ~[?:?] |  
  | »  	at java.security.KeyStore.load(KeyStore.java:1479) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig.getStore(TrustConfig.java:97) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.StoreKeyConfig.createTrustManager(StoreKeyConfig.java:94) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig$CombiningTrustConfig.lambda$createTrustManager$0(TrustConfig.java:168) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:550) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:517) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.TrustConfig$CombiningTrustConfig.createTrustManager(TrustConfig.java:169) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.createSslContext(SSLService.java:442) ~[?:?] |  
  | »  	at java.util.HashMap.computeIfAbsent(HashMap.java:1133) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.lambda$loadSSLConfigurations$5(SSLService.java:531) ~[?:?] |  
  | »  	at java.util.HashMap.forEach(HashMap.java:1336) ~[?:?] |  
  | »  	at java.util.Collections$UnmodifiableMap.forEach(Collections.java:1505) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.loadSSLConfigurations(SSLService.java:529) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.ssl.SSLService.<init>(SSLService.java:140) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createSSLService(XPackPlugin.java:413) ~[?:?] |  
  | »  	at org.elasticsearch.xpack.core.XPackPlugin.createComponents(XPackPlugin.java:263) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.lambda$new$15(Node.java:542) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:271) ~[?:?] |  
  | »  	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] |  
  | »  	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?] |  
  | »  	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] |  
  | »  	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:546) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.node.Node.<init>(Node.java:279) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:225) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:387) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:170) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
```

--- a/x-pack/plugin/sql/qa/jdbc/security/with-ssl/build.gradle
+++ b/x-pack/plugin/sql/qa/jdbc/security/with-ssl/build.gradle
@@ -10,8 +10,6 @@ testClusters.all {
 }
 
 // JDBC client can only be configured for SSL with keystores, but we can't use JKS/PKCS12 keystores in FIPS 140-2 mode.
-if (BuildParams.inFipsJvm) {
-  project.tasks.configureEach {
-    enabled = true
-  }
+tasks.withType(Test).configureEach {
+  onlyIf { BuildParams.inFipsJvm == false}
 }

--- a/x-pack/plugin/sql/qa/jdbc/security/with-ssl/build.gradle
+++ b/x-pack/plugin/sql/qa/jdbc/security/with-ssl/build.gradle
@@ -1,3 +1,5 @@
+import org.elasticsearch.gradle.info.BuildParams
+
 apply plugin: 'elasticsearch.test-with-ssl'
 
 testClusters.all {
@@ -5,4 +7,11 @@ testClusters.all {
   setting 'xpack.license.self_generated.type', 'trial'
   setting 'xpack.security.http.ssl.enabled', 'true'
   setting 'xpack.security.transport.ssl.enabled', 'true'
+}
+
+// JDBC client can only be configured for SSL with keystores, but we can't use JKS/PKCS12 keystores in FIPS 140-2 mode.
+if (BuildParams.inFipsJvm) {
+  project.tasks.configureEach {
+    enabled = true
+  }
 }

